project/Dependencies.scala

```diff
@@ -26,7 +26,7 @@ object Dependencies {
   val workbenchOpenTelemetryV = "0.1-e66171c"
   val workbenchErrorReportingV = "0.1-92fcd96"
 
-  val helmScalaSdkV = "0.0.1-RC1"
+  val helmScalaSdkV = "0.0.1-rt-uninstall-SNAPSHOT"
```

Comment:
TODO: change back to 0.0.1-RC1 once https://github.com/broadinstitute/helm-scala-sdk/pull/5 is merged